lib/logthrdestdrv.c

```diff
@@ -329,6 +352,9 @@ _perform_inserts(LogThreadedDestWorker *self)
 
       _process_result(self, result);
 
+      if (self->enable_batching && self->batch_size == self->owner->batch_lines)
```

Comment:
shouldn't we check `self->batch_size >= self->owner->batch_lines`?

Reply:
fixed thanks